src/test/java/net/snowflake/client/jdbc/telemetryOOB/TelemetryServiceIT.java

```diff
@@ -35,7 +36,9 @@ public class TelemetryServiceIT extends BaseJDBCTest {
   @Before
   public void setUp() {
     TelemetryService service = TelemetryService.getInstance();
-    service.updateContextForIT(getConnectionParameters());
+    Map<String, String> connectionParams = getConnectionParameters();
+    connectionParams.put("TELEMETRYDEPLOYMENT", "K8TEST");
```

Comment:
in telemetry tests we should always use `K8TEST`? on local tests should not we use e.g. REG or DEV?

Reply:
Hey Dominik, Irrespective of whether we use REG / DEV we were really always going to `sfcdev` as the Telemetry deployment URL. I am adding this specifically only for this test so that we continue to have tests for traditional stuff. Once we move over to the new K8s based telemetry deployment we can change the this to REG / DEV / PROD etc.